eng/versioning/version_client.txt

```diff
@@ -34,7 +34,7 @@ com.azure:azure-data-appconfiguration;1.1.6;1.2.0-beta.1
 com.azure:azure-data-schemaregistry;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-schemaregistry-avro;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-tables;12.0.0-beta.2;12.0.0-beta.3
-com.azure:azure-digitaltwins-core;1.0.0-beta.3;1.0.0
+com.azure:azure-digitaltwins-core;1.0.0;1.0.1
```

Comment:
If this hasn't been released GA yet the dependency version should remain the last beta and the shipping version being the current version.

Reply:
Once it is shipped an automated PR will be submitted to update the dependency version to `1.0.0` and the current version to `1.1.0-beta.1`.